src/rules/no-nodejs-modules.js

```diff
@@ -14,6 +14,17 @@ module.exports = {
     docs: {
       url: docsUrl('no-nodejs-modules'),
     },
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          allow: {
+            type: 'array',
```

Comment:
what is the type of the items? must they be unique?

Reply:
Good catch, added those checks.